Frameworks/Foundation/NSOperation.mm

```diff
@@ -131,139 +128,109 @@ - (void)setThreadPriority:(double)priority {
  @Status Interoperable
 */
 - (void)setCompletionBlock:(void (^)(void))block {
-    id oldBlock = priv->completionBlock;
-    priv->completionBlock = [block copy];
-    [oldBlock release];
+    pthread_mutex_lock(&_completionBlockLock);
+    [_completionBlock autorelease];
```

Comment:
I think it's unnecessary. I'll change back to release.

---
In reply to: [69351630](https://github.com/Microsoft/WinObjC/pull/602#discussion_r69351630) [](ancestors = 69351630)